stages/org.osbuild.ostree.encapsulate

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/python3
+"""
+Wrap OSTree commits into a oci container image
+
+For more information see [1].
+
+[1] https://github.com/ostreedev/ostree-rs-ext#module-container-bridging-between-ostree-and-ocidocker-images
+"""
+
+import subprocess
+import sys
+
+import osbuild.api
+
+CAPABILITIES = ["CAP_MAC_ADMIN"]
+
+
+SCHEMA_2 = """
+"options": {
+  "additionalProperties": false,
+  "required": ["filename"],
+  "properties": {
+    "filename": {
+      "description": "Resulting image filename",
+      "type": "string"
+    },
+    "cmd": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "copymeta": {
+      "description": "Propagate an OSTree commit metadata key to container label",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "format_version": {
+      "type": "integer",
+      "description": "The encapsulated container format version",
+      "default": 1
+    },
+    "labels": {
+      "description": "Additional labels for the container",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "max_layers": {
+      "description": "Maximum number of container image layers",
+      "type": "integer"
+    }
+  }
+},
+"inputs": {
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["commit"],
+  "properties": {
+    "commit": {
+      "type": "object",
+      "additionalProperties": true
+    }
+  }
+}
+"""
+
+
+def ostree(*args, _input=None, **kwargs):
+    args = list(args) + [f'--{k}={v}' for k, v in kwargs.items()]
+    print("rpm-ostree " + " ".join(args), file=sys.stderr)
+    subprocess.run(["rpm-ostree"] + args,
+                   encoding="utf-8",
+                   stdout=sys.stderr,
+                   input=_input,
+                   check=True)
+
+
+def parse_input(inputs):
+    commits = inputs["commit"]
+    data = commits["data"]
+    refs = data["refs"]
+    assert len(refs) == 1, "Need exactly one commit"
+    commit_id = next(iter(refs.keys()))
```

Comment:
This could be written as: `commit_id, = refs.keys()` which will unpack `refs.keys()` into `commit_id` and also make the above assert unnecessary (ValueError would be raised if more values are there).